src/components/Country/Country.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { getAllCountries, getRegion } from '../../redux/countriesSlice/countriesSlice';
+import './Country.scss';
+
+const Country = () => {
+  const {
+    countriesData, isLoading, region,
+  } = useSelector(
+    (store) => store.country,
+  );
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!countriesData.length) dispatch(getAllCountries());
+    if (region) dispatch(getRegion(region));
+  }, [dispatch, countriesData.length, region]);
+
+  return (
+    <div className="country__list">
+      {isLoading ? (
+        <h1>Loading...</h1>
+      ) : (
+        countriesData.length > 0
+        && countriesData.map((item) => (
+          <Link to={`/${item.cioc}`} key={item.name.common} className="country__container">
+            <img src={item.flags.png} alt={item.flags.alt} />
+            <div className="country__content">
+              <BsArrowRightCircle className="country__arrow" />
+              <h3>
+                Name:
+                {' '}
+                {item.name.common}
+              </h3>
+              <p>
+                Capital:
+                {' '}
+                {item.capital}
+              </p>
+              <p>
+                Population:
+                {' '}
+                {item.population}
+              </p>
+            </div>
+          </Link>
+        ))
+      )}
+    </div>
```

Comment:
- [ ]  It would be better to display two cards in a row, this will make your design look more like the Behance design, this will also help take less space on the home page, Great work so far 👏🏿 


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_788](https://user-images.githubusercontent.com/69638013/227608099-3b45a793-84e3-40d5-acdd-39d28b0339e5.png) | ![Screenshot_787](https://user-images.githubusercontent.com/69638013/227607945-d08b47b8-5d06-4d3d-a108-94e3cfe7aba2.png)